src/integrations.ts

```diff
@@ -0,0 +1,330 @@
+export type Tag = "CMS" | "E-commerce" | "Library" | "Mobile";
+
+export interface Integration {
+  name: string;
+  slug: string;
+  tags: Tag[];
+  fcVersion: "v1" | "v2";
+  image: string;
+  link: string;
+  github?: string;
+  source?: string;
+  official?: boolean;
+}
+
+export const INTEGRATIONS: Integration[] = [
+
+  // E-commerce plugins
+  {
+    name: "Magento 2",
+    slug: "magento-2",
+    tags: ["E-commerce"],
+    fcVersion: "v1",
+    image: "magento-2.svg",
+    link: "https://github.com/iMi-digital/magento2-friendly-captcha",
+    github: "iMi-digital/magento2-friendly-captcha",
+  },
+  {
+    name: "Adobe Commerce",
+    slug: "adobe-commerce",
+    tags: ["E-commerce"],
+    fcVersion: "v1",
+    image: "adobe-commerce.svg",
+    link: "https://github.com/iMi-digital/magento2-friendly-captcha",
+    github: "iMi-digital/magento2-friendly-captcha",
+  },
+  {
+    name: "Shopware",
+    slug: "shopware",
+    tags: ["E-commerce"],
+    fcVersion: "v1",
+    image: "shopware.svg",
+    link: "https://store.shopware.com/en/laude19210865287/friendly-captcha.html",
+  },
+
+  // CMS plugins
+  {
+    name: "Contao",
+    slug: "contao",
+    tags: ["CMS"],
+    fcVersion: "v1",
+    image: "contao.svg",
+    link: "https://github.com/FriendlyCaptcha/friendly-captcha-contao",
+    github: "FriendlyCaptcha/friendly-captcha-contao",
+    official: true,
+  },
+  {
+    name: "Craft CMS",
+    slug: "craft",
+    tags: ["CMS"],
+    fcVersion: "v1",
+    image: "craft.svg",
+    link: "https://github.com/digitalpulsebe/craft-friendly-captcha",
+    github: "digitalpulsebe/craft-friendly-captcha",
+  },
+  {
+    name: "Drupal",
+    slug: "drupal",
+    tags: ["CMS"],
+    fcVersion: "v1",
+    image: "drupal.svg",
+    link: "https://www.drupal.org/project/friendlycaptcha",
+    source: "https://git.drupalcode.org/project/friendlycaptcha",
+  },
+  {
+    name: "Joomla",
+    slug: "joomla",
+    tags: ["CMS"],
+    fcVersion: "v1",
+    image: "joomla.svg",
+    link: "https://github.com/SharkyKZ/plg_captcha_friendlycaptcha",
+    github: "SharkyKZ/plg_captcha_friendlycaptcha",
+  },
+  {
+    name: "TYPO3",
+    slug: "typo3",
+    tags: ["CMS"],
+    fcVersion: "v1",
+    image: "typo3.svg",
+    link: "https://github.com/studiomitte/friendlycaptcha-typo3",
+    github: "studiomitte/friendlycaptcha-typo3",
+  },
+
+  // Libraries and SDKs
+  {
+    name: "Go",
+    slug: "golang",
+    tags: ["Library"],
+    fcVersion: "v2",
+    image: "golang.svg",
+    link: "https://github.com/friendlycaptcha/friendly-captcha-go",
+    github: "FriendlyCaptcha/friendly-captcha-go",
+    official: true,
+  },
+  {
+    name: "Go",
+    slug: "v1/golang",
+    tags: ["Library"],
+    fcVersion: "v1",
+    image: "golang.svg",
+    link: "https://github.com/friendlycaptcha/friendly-captcha-go-sdk",
+    github: "FriendlyCaptcha/friendly-captcha-go-sdk",
+    official: true,
```

Comment:
NIT: I guess `official` could be a tag as well?

Reply:
I sort of like that the attributes that the keys that create badges on the cards are separate from the list of tags, which just establish the category filters